nextflow.config

```diff
@@ -38,6 +38,28 @@ profiles {
     process.container = 'quay.io/nextflow/rnaseq-nf:v1.1'
     docker.enabled = true
   }
+
+  singularity {
+    process.container = 'quay.io/nextflow/rnaseq-nf:v1.1'
```

Comment:
Can we not just define this once, at the top scope?

Reply:
I did think about that too but I assumed they were added for a reason. For example, isolating the `container` directive when `conda` is being used.

Reply:
I think it's just ignored when conda is being used. [This is the DSL1 setup](https://github.com/nf-core/methylseq/blob/03972a686bedeb2920803cd575f4d671e9135af0/nextflow.config#L87-L89) that we used to have in nf-core and it worked fine.

Reply:
You are right, it is ignored but it may be better practice not to have to ignore it and set in in the correct locations?